CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
+documentation, we greatly value feedback and contributions from our community.
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+
```

Comment:
Moving security notice up to the top and replacing with revised wording per @juneb's edits in https://github.com/aws/aws-encryption-sdk-python/blob/master/CONTRIBUTING.md
```suggestion

## Security issue notifications
If you discover a potential security issue in this project, notify AWS/Amazon Security by using our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public GitHub issue.

```